pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -179899,14 +179899,13 @@ self: {
          base cereal lens megaparsec mtl optparse-applicative resourcet rio
          template-haskell time unix unliftio
        ];
+       libraryToolDepends = [ pkgs.git ];
```

Comment:
Did you add this line manually?

This file is automatically generated, so any manual additions will get overwritten.  You'll have to add this override to `pkgs/development/haskell-modules/configuration-common.nix`. (Grepping through the file for the string "tool" should hopefully give you some examples of how to do this)

Reply:
I did add that line manually. Sorry, I don't really know how things work yet—very new to nix—and am just waddling around in the darkness :)

I'll push a fix